tests/pending/pos/depmet_implicit_oopsla_session_2.scala

```diff
@@ -18,7 +17,7 @@ object Sessions {
 
   // friendly interface to the theory
   def runSession[S, D: Session[S]#HasDual](session: S, dual: D) =
-    ?[Session[S]#HasDual[D]].run(session, dual)
+    implicitly[Session[S]#HasDual[D]].run(session, dual)
```

Comment:
Was keeping this test pending intentional? Does this PR change how `implicitly` deals with type member?

I remember seeing something similar to this `def ?` being use to get more precise types:

```scala
trait A { type Out }

implicit val a = new A { type Out = String }

val i1: A { type Out = String } = implicitly[A]
// error: type mismatch;
// found   : A
// required: A{type Out = String}

def ?[T <: AnyRef](implicit w: T): w.type = w

val i2: A { type Out = String } = ?[A]  // works fine!
```